README.md

```diff
@@ -91,14 +91,22 @@ You only need to run `dap-dart-setup` one time to setup the debugger to your Ema
 You can register a custom template for debugging with `dap-register-debug-template`, check the following example:
 
 ```elisp
-(use-package lsp-dart 
-  :ensure t 
+(use-package lsp-dart
+  :ensure t
   :hook (dart-mode . lsp)
   :init
   (dap-register-debug-template "Flutter :: Custom debug"
                                (list :flutterPlatform "x86_64"
-                                     :args '("--flavor" "staging"))))
+                                     :program "lib/main_debug.dart"
+                                     :args '("--flavor" "customer_a"))))
 ```
+Both `:program` and `:args` are optional.
```

Comment:
Actually `flutterPlatform` is optional too because already uses the default: `default`.
In the latest release I made all the important args set by default, that's why all params of the list are optional :)

Reply:
Good to know. Fixed.